Add a guide for economic buyers

ptgott opened the pull request:
Introduce the three Teleport products, spell out use cases, and provide a very brief architectural overview. In each section that describes a product, include links to relevant docs guides.

The plan is to feature this guide more centrally once we reorganize the docs. For now, this change adds the guide to the docs without including navigation links to make it possible to review the text.

ptgott committed:
Add a guide for economic buyers

Introduce the three Teleport products, spell out use cases, and provide
a very brief architectural overview. In each section that describes a
product, include links to relevant docs guides.

The plan is to feature this guide more centrally once we reorganize the
docs. For now, this change adds the guide to the docs without including
navigation links to make it possible to review the text.

xinding33 commented on docs/pages/what-is-teleport.mdx:
```diff
@@ -0,0 +1,108 @@
+protected behind a firewall. 
+
+All infrastructure resources belong to a unified inventory, with a single
+role-based access controls system to allow for least-privilege access. You can
+enable users to authenticate to resources using Single Sign-On providers like
+GitHub and Google Workspace. Connectivity takes place through short-lived
```
It'd be good to clarify that Teleport can act as the SSO or you can integrate your existing SSO. In terms of SSOs to mention, I think it's important to include Okta as that's what a lot of folks will be specifically looking for.

xinding33 commented on docs/pages/what-is-teleport.mdx:
```diff
@@ -0,0 +1,108 @@
+
+Get started with Teleport Access by enrolling resources in your cluster:
+
+- [Servers](server-access/introduction.mdx)
+- [Databases](database-access/introduction.mdx)
+- [Kubernetes clusters](kubernetes-access/introduction.mdx)
+- [Windows desktops](desktop-access/introduction.mdx)
+- [Applications](application-access/introduction.mdx)
```
I think "enrolling resources" should be a single section (something like "unified access to all resources"). Other sections should include other benefits of Teleport Access: cryptographic identity, audit logs / session recording, etc.

xinding33 commented on docs/pages/what-is-teleport.mdx:
```diff
@@ -0,0 +1,108 @@
+infrastructure. With Teleport Access Graph, you can get insight into role-based
+access control policies, including in Teleport and your cloud provider. 
+
+Get started with [Teleport Access Graph](access-controls/access-graph.mdx).
```
Would link this to the new end-to-end policy guide.

ptgott replied:
This will take a little more work than I anticipated, so I'll plan to decouple the end-to-end Access Graph guide project from the one to make the buyer guide the home page and get it done ASAP.

xinding33 reviewed (approved):
I like the general direction with the product split and architecture overview. For the individual product sections, I suggest focusing on the benefits that said product provides or problems that said product solves rather than feature-by-feature. But overall, I'd say this is much better than what we currently have (i.e. no buyer guide) so let's just ship it and iterate from there.

ptgott committed:
Turn the economic buyer guide into the home page

Also respond to xinding33 feedback.

The economic buyer guide will serve as the home page until we have a
more purpose-built home page that provides information on the three main
Teleport products.

Also edit the introduction to the "Teleport Agents" section to make it
more immediately useful as a guide to enrolling resources with Teleport
that we can link to from the "Teleport Access" discussion in the
economic buyer guide.

Moves the original home page to "Deploy a Cluster" as a guide to
deploying a demo cluster.

ptgott commented:
**Note:** Incorporate messaging edits from Marketing before backporting. Backport along with the messaging edits.

travelton commented on docs/pages/index.mdx:
```diff
@@ -1,221 +1,124 @@
+
+### Teleport Policy
+
+**Teleport Policy** unifies and controls access policies across all of your
+infrastructure. With Teleport Access Graph, you can get insight into role-based
+access control policies, including in Teleport and your cloud provider. 
```
Minor nit to improve readability...

```suggestion
**Teleport Policy** unifies and controls access policies across all your infrastructure. With Teleport Access Graph, you gain insights into role-based access control policies within Teleport and your cloud provider.
```

travelton commented on docs/pages/index.mdx:
```diff
@@ -1,221 +1,124 @@
+
+Teleport Agents, which the user deploys on Linux servers or Kubernetes, proxy
+access to infrastructure resources and cloud provider APIs. Users authenticate
+to infrastructure resources through Teleport agents using short-lived
```
```suggestion
to infrastructure resources through Teleport Agents using short-lived
```

travelton commented on docs/pages/index.mdx:
```diff
@@ -1,221 +1,124 @@
+Teleport Agents, which the user deploys on Linux servers or Kubernetes, proxy
+access to infrastructure resources and cloud provider APIs. Users authenticate
+to infrastructure resources through Teleport agents using short-lived
+certificates. Certificates indicate Teleport role membership, allowing Teleport
```
```suggestion
certificates. Certificates embed Teleport user attributes, allowing Teleport
```

travelton commented on docs/pages/index.mdx:
```diff
@@ -1,221 +1,124 @@
+access to infrastructure resources and cloud provider APIs. Users authenticate
+to infrastructure resources through Teleport agents using short-lived
+certificates. Certificates indicate Teleport role membership, allowing Teleport
+Agents to enforce role-based access-controls.
```
```suggestion
Agents to enforce role-based access controls.
```

travelton commented on docs/pages/index.mdx:
```diff
@@ -1,221 +1,124 @@
+identity-aware access proxy that run either on the Teleport-managed cloud or, in
+special cases, a self-hosted private network. 
+
+Teleport Agents, which the user deploys on Linux servers or Kubernetes, proxy
```
```suggestion
Teleport Agents, which are deployed on Linux servers or Kubernetes clusters, proxy
```

ptgott replied:
I've suggested "can run" to avoid raising questions of how the deployment works

travelton commented on docs/pages/index.mdx:
```diff
@@ -1,221 +1,124 @@
+
+**Teleport Identity** is an add-on to Teleport Access that offers identity
+governance and security for all of your infrastructure. You can provide users
+less-privileged roles by default, requiring any user who wants additional
```
```suggestion
with less-privileged roles by default, requiring any user who wants additional
```

travelton commented on docs/pages/index.mdx:
```diff
@@ -1,221 +1,124 @@
-title: Get Started with Teleport
-description: This tutorial will guide you through the steps needed to install and run Teleport on a Linux server
-videoBanner: BJWbSqiDLeU
+title: Introduction to Teleport
+description: Provides an overview of the purpose and benefits of Teleport Access Platform, which allows you to implement Zero Trust for all of your infrastructure.
```
```suggestion
title: Introduction to the Teleport Access Platform: Zero Trust Security for Your Infrastructure
description: Explore an overview of the Teleport Access Platform. Learn how to implement Zero Trust Security across all your infrastructure for enhanced protection and streamlined access control.
```
SEO optimized

ptgott replied:
"Explore" seemed like too strong a word here. I've suggested "Read".

travelton commented on docs/config.json:
```diff
@@ -5,7 +5,7 @@
       "entries": [
         {
-          "title": "Get Started with Teleport",
+          "title": "Introduction to Teleport",
```
```suggestion
          "title": "Introduction to the Teleport Access Platform: Zero Trust Security for Your Infrastructure",
```
SEO optimization

travelton replied:
Ah! Sorry, not super familiar with how the docs work. Makes sense.

travelton commented on docs/pages/agents/introduction.mdx:
```diff
@@ -1,19 +1,40 @@
-title: "Teleport Agents"
-description: Deploy agents to enroll resources in your infrastructure with Teleport. You can run multiple Teleport services per agent."
+title: Protect Infrastructure with Teleport
+description: Deploy Agents to enroll resources in your infrastructure with Teleport. You can run multiple Teleport services per Agent."
```
```suggestion
description: Deploy Agents to enroll resources in your infrastructure with Teleport. You can run multiple Teleport services per Agent.
```
Is this supposed to be there?

travelton commented on docs/pages/agents/introduction.mdx:
```diff
@@ -29,85 +50,65 @@ services are enabled by default and how to enable a particular service.
+In this setup, Agents dial the Teleport Proxy Service in order to establish
 reverse SSH tunnels. While the Proxy Service remains open to the public internet
-via its HTTPS port, agents require no open ports or public address.
+via its HTTPS port, Agents require no open ports or public address.
```
```suggestion
via its HTTPS port, Agents require only outbound network access to the Proxy Service.
```
This is a little misleading... And I know it wasn't part of the original change. However, this could be clarified with this change.

ptgott replied:
The change made it less explicit that you can run Agents behind a firewall, so I've incorporated this suggestion and added that language as well.

travelton commented on docs/pages/deploy-a-cluster/linux-demo.mdx:
```diff
@@ -0,0 +1,208 @@
+---
+
+See how a self-hosted Teleport deployment works by completing the tutorial
+below. This shows you how to spin up a single-instance Teleport cluster on a
```
```suggestion
below. This shows you how to spin up a single instance Teleport cluster on a
```

ptgott replied:
This is a compound modifier, so I think the hyphen is okay here.

travelton commented on docs/pages/deploy-a-cluster/linux-demo.mdx:
```diff
@@ -0,0 +1,208 @@
+
+## Next step: deploy agents
+
+Teleport **agents** proxy traffic to infrastructure resources like servers,
```
```suggestion
Teleport **Agents** proxy traffic to infrastructure resources like servers,
```

travelton reviewed (approved):
A few things, but approving so you can move forward.
